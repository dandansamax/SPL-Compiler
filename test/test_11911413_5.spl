
int test_3 int a,int b{
    if a>b{

    }else if b>a{

    }
}
int test_11911413_5(){


   return 0;
}