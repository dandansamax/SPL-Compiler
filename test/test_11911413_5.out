Error type B at Line 2: Missing closing parenthesis '(' and ')'
Error type B at Line 5: Missing closing parenthesis '(' and ')'
Error type B at Line 3: Missing closing parenthesis '(' and ')'
